network/p2p/pubsub.go

```diff
@@ -53,6 +53,8 @@ const (
 // TXTopicName defines a pubsub topic for TX messages
 const TXTopicName = "/algo/tx/0.1.0"
 
+const incomingThreads = 20 // matches to number wsNetwork workers
```

Comment:
Good there is a comment here, may want to put an equivalent one on wsNetwork workers if not there already

Reply:
(Better if easy for them to use a common config param, even if hard-coded)

Reply:
I suggest to do it after the merge by refactoring parameters setting in NewP2PNetwork, NewWsNetwork and MakeService. This way it will be possible to pass numIncomingWorkers from network package down to p2p's MakeService without overgrowing an ordered MakeService parameters list.